apps/src/music/blockly/MusicBlocklyWorkspace.ts

```diff
@@ -39,7 +39,7 @@ export default class MusicBlocklyWorkspace {
 
   // Utility to hide any custom fields that are showing.
   public static hideChaff() {
-    Blockly.getMainWorkspace().hideChaff();
+    Blockly.getMainWorkspace()?.hideChaff();
```

Comment:
Would it be possible to make this non-static and do `this.workspace?.hideChaff()`?  That way it's more closely associated with the current workspace instance?

Reply:
Fair call.  Updated.